package.json

```diff
@@ -146,6 +146,7 @@
     "react-native-default-preference": "1.4.1",
     "react-native-device-info": "^6.0.2",
     "react-native-elements": "^1.2.7",
+    "react-native-exit-app": "^1.1.0",
```

Comment:
Actually, so far we used RN module BackHandler.exitApp(); to close the app. Is there any special need to add this library in here?

Reply:
yes

Reply:
your solution doesn't work in a modal view see warnings https://reactnative.dev/docs/backhandler

Reply:
done with out changes